docs/study-view/simple-filter.md

```diff
@@ -0,0 +1,11 @@
+## Simple Filter
+
+The study view support pre-filtered values of one attribute in URL.
+
+### How to
+Two parameters should be specified in the URL:
+
+* filterAttributeId: the clinical attribute id (we only accept one attribute id). Please refer to clinical_attribute_meta table, ATTR_ID column
```

Comment:
We need to find a way to expose this. Maybe we can have it in the mouse over of chart header?